### ISSUE TITLE
Heketi: Error: signature is invalid

### ISSUE THREAD

**Issue by bigon** [openshift/openshift-ansible#9068]
#### Description

When running the deploy_cluster.yml playbook on an already installed cluster it fails at the "Verify heketi service" step

##### Version

```
openshift-ansible-3.9.31-1.git.34.154617d.el7.noarch
```

##### Steps To Reproduce
1. ansible-playbook -i tst-inventory.ini  /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
2. ansible-playbook -i tst-inventory.ini  /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml

##### Expected Results
The playbook succeed on the 2nd time as well

##### Observed Results
Describe what is actually happening.

```
TASK [openshift_storage_glusterfs : Verify heketi service] ********************************************************************************************************************************************************
fatal: [master1.example.com]: FAILED! => {"changed": false, "cmd": ["oc", "--config=/tmp/openshift-glusterfs-ansible-IHz1ry/admin.kubeconfig", "rsh", "--namespace=glusterfs", "heketi-storage-1-n68t6", "heketi-cli", "-s", "http://localhost:8080", "--user", "admin", "--secret", "2uHQXxBvIPeRknQJ/giepFogu3tlxyXpfaviVkhFVgs=", "cluster", "list"], "delta": "0:00:00.480113", "end": "2018-07-04 13:59:40.023812", "msg": "non-zero return code", "rc": 255, "start": "2018-07-04 13:59:39.543699", "stderr": "Error: signature is invalid\ncommand terminated with exit code 255", "stderr_lines": ["Error: signature is invalid", "command terminated with exit code 255"], "stdout": "", "stdout_lines": []}
```

For long output or logs, consider using a [gist](https://gist.github.com/)

##### Additional Information

```
[...]
[OSEv3:vars]
openshift_storage_glusterfs_is_native=false
openshift_storage_glusterfs_storageclass=true
openshift_storage_glusterfs_heketi_is_native=true
openshift_storage_glusterfs_heketi_executor=ssh
openshift_storage_glusterfs_heketi_ssh_port=22
openshift_storage_glusterfs_heketi_ssh_user=ansible
openshift_storage_glusterfs_heketi_ssh_sudo=true
openshift_storage_glusterfs_heketi_ssh_keyfile="/root/.ssh/id_rsa"
[...]
[glusterfs]
node1.example.com glusterfs_ip=10.122.92.39 glusterfs_devices='[ "/dev/sdc" ]'
node2.example.com glusterfs_ip=10.122.92.41 glusterfs_devices='[ "/dev/sdc" ]'
node3.example.com glusterfs_ip=10.122.92.43 glusterfs_devices='[ "/dev/sdc" ]'
```

**Comment by bigon** [comment]
Nobody? The lack of impotency in the playbook is quite annoying...

**Comment by bigon** [comment]
OK so this is fixed by 6609b6cce62353348a4a1e1736d3ec70d7610ca6 in the opensource version. Still an issue in the enterprise one as the RPM packages are not updated yet

**Comment by patlachance** [comment]
A [bugzilla case](https://bugzilla.redhat.com/show_bug.cgi?id=1637105) referencing this issue was opened.
